packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -300,6 +301,8 @@ export default function DataviewsPatterns() {
 				header: __( 'Sync status' ),
 				id: 'sync-status',
 				render: ( { item } ) => {
+					const syncStatus =
+						item.wp_pattern_sync_status || PATTERN_SYNC_TYPES.full;
```

Comment:
It seems we need to check if it's a user pattern, [similarly](https://github.com/WordPress/gutenberg/pull/63042/files#diff-0f49c90dc39786ecdf45f374247e433f763d295d80bdaecda3bab11357e4d7fdR147) with the search.

Reply:
If you see a few lines above, this field is only added for "user" patterns.

Reply:
Will check now, but the rendering of the label is wrong for patterns..

Reply:
I pushed a [fix](https://github.com/WordPress/gutenberg/pull/63042/commits/0d6d391f43d3830c5464127a10d1d7aaa15df703) for this, but feel free to revert/update if you have a better solution..